bigbluebutton-html5/tests/webdriverio/pageobjects/landing.page.js

```diff
@@ -16,25 +16,45 @@ class LandingPage extends Page {
     return 'http://localhost:8080/demo/demoHTML5.jsp';
```

Comment:
Yeah, you are right. And we don't even need to put it into a config, the base URL already exists in the test runner config: https://github.com/MaximKhlobystov/bigbluebutton/blob/397a94b1c937cc72b48b6aa96d03126d1ffff3ce/bigbluebutton-html5/tests/webdriverio/wdio.conf.js#L15. I just made it exposed: https://github.com/MaximKhlobystov/bigbluebutton/blob/397a94b1c937cc72b48b6aa96d03126d1ffff3ce/bigbluebutton-html5/tests/webdriverio/wdio.conf.js#L32 and started using it: https://github.com/MaximKhlobystov/bigbluebutton/blob/397a94b1c937cc72b48b6aa96d03126d1ffff3ce/bigbluebutton-html5/tests/webdriverio/pageobjects/landing.page.js#L17.